app/views/granules/download_links.html.erb

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title><%= site_page_title('Download Granule Links') %></title>
+  <link rel="shortcut icon" href="/favicon.ico?v=1.1">
+  <style>
```

Comment:
You need to separate the style of this page out to it's own SCSS file. We always try to avoid straight CSS in a `style` element.

Reply:
@mreese84 - Please take a look at the other erb files within the same directory.  There is app/views/granules/download.html.erb and app/views/granules/prepare_script.html.erb. If you look at those, then you'll see why I did it this way - inline CSS is actually the established method within these little download mechanism erb files.   

This solution was also picked as it is consistent - this is how the other download mechanisms are designed.  Both of those other files have the same inline CSS structure; actually, prepare_script.html.erb was used as a template to generate this new file.  With this mechanism the new download_links has consistent behavior with the other download methods.

This kind of answers your other question as well - having the button on the front page turn into a spinner while it is all working on the backend would be a new design strategy.  My issues with it are that - first, my gut tells me that a user is more likely to feel impatient with that strategy (clicking a button and see it turn disabled while it spins for 60 seconds is oddly frustrating - a page refresh would be less so, to me).  Second, it would be a new design approach differentiating from the established method of the erb files, and so would take some thought to do (days, at least).  So, to me, that would be little to no gain for a lot of work.

Reply:
Ok. I buy it as that's the established pattern. I don't necessarily like the pattern, but it is what it is and to establish/redesign would've been a larger ticket altogether. So... approved on my end. Please wait for one other approval before merging.